src/main/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/model/prison/prisonersearch/PrisonerSearchService.kt

```diff
@@ -31,6 +32,9 @@ class PrisonerSearchService(@Autowired private val client: PrisonerSearchApiClie
   private fun matchPrisoner(identifier: String): List<MatchPrisonerResponse> =
     client.matchPrisoner(MatchPrisonerRequest(identifier))
 
+  fun getPrisoner(prisonNumber: String): Prisoner =
+    client.getPrisoner(MatchPrisonerRequest(prisonNumber)).component1()
```

Comment:
need to throw not found if absent, this will throw a 500 if missing.  
```
client.getPrisoner(MatchPrisonerRequest(prisonNumber))

```

Reply:
okay